qiskit/algorithms/state_fidelities/compute_uncompute.py

```diff
@@ -66,6 +81,7 @@ def __init__(self, sampler: BaseSampler, options: Options | None = None) -> None
                 f"The sampler should be an instance of BaseSampler, " f"but got {type(sampler)}"
             )
         self._sampler: BaseSampler = sampler
+        self._average_local = local
```

Comment:
Maybe it would be better to call this `self._local` too 🙂

Reply:
thanks, totally missed that